content/go/concepts/operators/operators.md

```diff
@@ -0,0 +1,106 @@
+- `++` Takes one expression, `(A++)`, and returns the expression incremented by one.
+- `--` Takes one expression, `(B--)`, and returns the expression decremented by one.
+
+## Comparison Operators
+
+Comparison operators compare two expressions and return `true` or `false` based on the operator and the values of the expression. Go has the following comparison operators.
+
+- `==`: Takes two expressions, `(A == B)`, and returns `true` if both expressions are equal.
+- `!=`: Takes two expressions, `(A != B)`, and returns `true` if both expressions are not equal.
+- `<`: Takes two expressions, `(A < B)`, and returns `true` if the left expression is numerically less than or comes lexically before the right one.
+- `<=`: Takes two expressions, `(A <= B)`, and returns `true` if the left expression is numerically less than or comes lexically before the right one, or if both expressions are equal.
+- `>`: Takes two expressions, `(A > B)`, and returns `true` if the left expression is numerically greater than or comes lexically after the right one.
+- `>=`: Takes two expressions, `(A >= B)`, and returns `true` if the left expression is numerically greater than or comes lexically after the right one, or if both expressions are equal.
+
+## Logical Operators
+
+Logical operators return a `true` or `false` value based on the `true` or `false` values of the expressions they operate on. When expressions are numeric, a zero value is treated as `false` and a non-zero value is treated as `true`.
+
+- `&&` AND operator: Takes two expressions, `(A && B)`, and returns `true` if both expressions are `true`.
+- `||` OR operator: Takes two expressions, `(A || B)`, and returns `true` if one of them is `true`.
+- `!` NOT operator: Takes one expression, `!(A)`, and returns `true` if its value is `false`.
+
+## Bitwise Operators
+
+Bitwise operators manipulate the individual bits of their numeric expressions.
+
+- `&` Binary AND Operator: Takes two expressions, `(A & B)`, result will have a bit set if the corresponding bit in each expression is set.
+- `|` Binary OR Operator: Takes two expressions, `(A | B)`, result will have a bit set if either expression has the corresponding bit set.
+- `^` Binary XOR Operator: Takes two arguments, `(A ^ B)`, result wil have a bit set if only one corresponding bit is set between the two expressions.
+- `<<` Binary Left Shift Operator: Takes two arguments, `(A << B)`, result will be the first argument with its bits moved to the left the number of positions specified by the second.
+- `>>` Binary Right Shift Operator: Takes two arguments, `(A >> B)`, result will be the first argument with its bits moved to the right the number of positions specified by the second.
+
+## Assignment Operators
+
+Assignment operators store the value of an expression into a variable. The variable being assigned to is always on the left.
+
+- `=`: Assigns the value of the expression to the right to the variable on the left.
```

Comment:
@SSwiniarski @Dusch4593 @KTom101 For these list items with additional text, what do you think about using this format:

```
<details>
  <summary>Text Displayed</summary>
  Text displayed after opening up toggle
</details>
```

This is not a format we're used to using for our markdown content, but I recommended using it in PR [599](https://github.com/Codecademy/docs/pull/599#discussion_r825339513)